README.md

```diff
@@ -79,6 +79,25 @@ More advanced changes can take effect using the `run.d` system. Similar to the `
 - If script terminates with exit code of $SIGNAL_BUILD_STOP (99), this will signal the container to stop cleanly. This can be used for a multi-stage build process
 
 
+### Shutdown Behavior
+
+Sequence of events for a crashed supervised service: 
+
+1. `finish` script is executed
```

Comment:
Should we add this link: https://github.com/just-containers/s6-overlay#writing-an-optional-finish-script here?